tokio/src/runtime/mod.rs

```diff
@@ -312,6 +312,15 @@ cfg_rt! {
         blocking_pool: BlockingPool,
     }
 
+    /// The flavor of the `Runtime`.
+    #[derive(Debug, PartialEq)]
+    pub enum RuntimeFlavor {
```

Comment:
We must be able to add additional flavors in the future.
```suggestion
    /// The flavor of the `Runtime`.
    #[derive(Debug, PartialEq)]
    #[non_exhaustive]
    pub enum RuntimeFlavor {
```

Reply:
Oh, didn't know that existed. Added.